lib/pliny/helpers/versioning.rb

```diff
@@ -0,0 +1,21 @@
+module Pliny::Helpers
+  module Versioning
+
+    # A variant is a specially tagged version where new API endpoints can be
+    # experimented with. If a user passes a version into the `Accept` header
+    # like "3.env-vars", the `api_version` will be "3" and the `api_variant`
+    # will be "env-vars".
+    #
+    # By design, only a single variant can be specified at a time.
+    def api_variant
+      # set by Versioning middleware
+      env["HTTP_X_API_VARIANT"]
```

Comment:
Nevermind. b2a048a088b2963e8511923b31240e50e9af1b8c
I've used `api.version` instead of `pliny`, because pliny is the middleware, not the app name itself.